UM-USElab/gradecraft-development
# Nearing Proficiency Doesn't Tick Outcome

### Status
READY

### Description
"Nearing Proficiency" Learning Objectives were erroneously incrementing the Outcome progress. This PR will prevent that.

### Migrations
NO

### Steps to Test or Reproduce

```sh
git pull --prune
git checkout <feature_branch>
bundle; foreman start
```

Create assessments on Learning Objectives - one with Proficient, one with Near Proficient. The Proficient LO should increment Outcome. The Near Proficient LO should not.

### Impacted Areas in Application
List general components of the application that this PR will affect:

* Learning Objectives
* Outcomes

======================
Closes #4116
